sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -257,31 +257,37 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
         """.stripMargin)
 
       // LOAD DATA INTO partitioned table must specify partition
-      withInputFile { path =>
+      withInputFile { f =>
         intercept[AnalysisException] {
+          val path = f.toURI.toString
```

Comment:
Just simply because some meet the limit of 100 line length in that case. I will try to clean up including the comment above.